docs/persistenceClustering1.md

```diff
@@ -0,0 +1,62 @@
+# Persistence Clustering 1 Example
+
+<!--[![Dragon example video tutorial](https://topology-tool-kit.github.io/img/gallery/dragon.jpg)](https://youtu.be/YVk9vRKIEX8)-->
+
+<!--iframe width="100%" height="420"
+src="https://www.youtube.com/embed/YVk9vRKIEX8" frameborder="0"
+allowfullscreen></iframe-->
+
+![Persistence Clustering 1 Image](https://topology-tool-kit.github.io/img/gallery/persistenceClustering1.jpeg)
+
+## Pipeline description
+<!--This example first loads a triangle mesh from disk.-->
+Firstly, this example loads a cloud of points from disk (top left view in the above screenshot), then computes a mesh on which a density field is obtained with a Gaussian Resampling on the points (top right view in the above screenshot) and will be considered as the input scalar data.
+<!--In a pre-processing, the mesh is smoothed and an elevation function is computed on top of it.-->
+<!--Then an elevation function is computed on it, and will be considered as the input scalar data for ou.-->
+
+Next, a [PersistenceDiagram](https://topology-tool-kit.github.io/doc/html/classttkPersistenceDiagram.html) is computed and thresholds are applied base on pair identifier to remove the diagonal, and on persistence to maintain only the features with a persistence above a certain value. The result is a simplified persistence diagram (bottom left view in the above screenshot).
```

Comment:
I think you can apply the threshold by persistence directly.
(the diagonal removal is just meant for the illustration)